r/react-zoom-pan-pinch/react_zoom_pan_pinch_rhel_7.6.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Package       : react-zoom-pan-pinch
+# Version       : commit #f78c546
+# Source        : https://github.com/prc5/react-zoom-pan-pinch.git
+# Tested on     : RHEL 7.6
+# Node Version  : v12.19.1
+# Maintainer    : Dhananjay Sathe <dhananjay.sathe@ibm.com>
+#
+# Disclaimer    : This script has been tested in non-root mode on given
+#                 platform using the mentioned version of the package.
+#                 It may not work as expected with newer versions of the
+#                 package and/or distribution. In such case, please
+#                 contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+set -e
+
+PACKAGE_VERSION=${1:-f78c546}
+
+#Install all dependencies.
+sudo yum clean all
+sudo yum -y update
```

Comment:
Are these really necessary? If not, please remove.

Reply:
@amitsadaphule These are not necessary. I removed it.